packages/tasit-action/src/testHelpers/helpers.js

```diff
@@ -56,16 +57,96 @@ export const confirmBalances = async (token, addresses, balances) => {
 // - Try to intercept and parse ethers.BigNumber to BN.js (and vice-versa)?
 // or
 // - Exposes ethers.utils.BigNumber / bigNumberify()?
-export const toBN = ethersBN => {
+const toBN = ethersBN => {
   return new BN(ethersBN.toString());
 };
 
-export default {
+const etherFaucet = async (
+  provider,
+  fromWallet,
+  beneficiaryAddress,
+  amountInWei
+) => {
+  const connectedFromWallet = fromWallet.connect(provider);
+  const tx = await connectedFromWallet.sendTransaction({
```

Comment:
Maybe put a comment in the code here with a TODO about doing it using `tasit-action` in the future

Reply:
Done.

Reply:
Thanks! Resolving